libs/config/angular/src/lib/config/config.service.ts

```diff
@@ -49,7 +10,6 @@ export class ConfigService {
   appUrl!: string;
   apiUrl!: string;
   email!: string;
-  docsUrl?: string;
```

Comment:
if the common hero-banner-util-button.component.html still refers to [href]="docsUrl", don't we still need it in each app's config?

Reply:
I have moved the setting logic out of the scope of `Config` and into a property of the shared `biosimulations-hero-util-button` component itself as such (.ts):

`public validateSimulatorUrl = 'https://docs.biosimulations.org';`
`public apiUrl = 'https://api.biosimulations.org';`
`public simulatorsApiUrl? = 'https://api.biosimulators.org';`
`public docsUrl = 'https://docs.biosimulations.org';`

and implemented in the component's html as such (for example):

<biosimulations-dropdown-menu-item heading="Validate a simulator" [href]="validateSimulatorUrl">
</biosimulations-dropdown-menu-item>


Since the `docsUrl` value is used ONLY for setting a value in the nav button on the hero banner and is the same for each app, this was done to prevent duplication.

Reply:
good answer, resolved!